app/models/tag.rb

```diff
@@ -7,8 +7,10 @@ def job_count
   end
 
   def average_salary
-    jobs.each do |job, sum|
-      (sum += job.salary) / job_count
+    sum = 0
+    self.jobs.each do |job|
+      sum = ((sum += job[:salary]) / job_count)
+    end
+    sum
```

Comment:
Is there a way we could calculate this without a Ruby enumerable? Perhaps by leveraging ActiveRecord?